ert_shared/main.py

```diff
@@ -418,6 +418,8 @@ def main():
             context.plugin_manager.add_logging_handle_to_root(logging.getLogger())
             logger.info("Running ert with {}".format(str(args)))
             args.func(args)
+    except ErtCliError as err:
+        sys.exit(str(err))
```

Comment:
Should we throw in a `logger.exception(str(err))` here?

Reply:
Maybe, but the reason I did not add it is because we would end up logging a bunch of things that are not errors, for example KeyboardInterupt.